romana-install/roles/romana/install-agent/tasks/service_configs_systemd.yml

```diff
@@ -4,3 +4,4 @@
   with_items:
     - dnsmasq-calico.service
     - romana-agent.service
+    - romana-policy-agent.service
```

Comment:
Does it work for openstack or only kubernetes?
The service name should reflect that.